lib/ecto/adapters/clickhouse.ex

```diff
@@ -119,7 +119,7 @@ defmodule Ecto.Adapters.ClickHouse do
       end
 
       @doc """
-      Similar to `Ecto.Repo.update_all/3` but uses [`ALTER TABLE ... UPDATE`](https://clickhouse.com/docs/en/sql-reference/statements/alter/update) instead.
+      Similar to Ecto.Repo.update_all/3 but uses [`ALTER TABLE ... UPDATE`](https://clickhouse.com/docs/en/sql-reference/statements/alter/update) instead.
```

Comment:
```suggestion
      Similar to `c:Ecto.Repo.update_all/3` but uses [`ALTER TABLE ... UPDATE`](https://clickhouse.com/docs/en/sql-reference/statements/alter/update) instead.
```
I believe this should work?

Reply:
very cool, didn't know about that!